packages/p2p/src/pages/app.jsx

```diff
@@ -128,6 +134,19 @@ const App = () => {
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, []);
 
+    React.useEffect(() => {
+        if (p2p_settings) {
```

Comment:
won't this condition be true even if p2p_settings is empty object, then the line `p2p_settings.currency_list.forEach` would give cannot read properties error

Reply:
ah yeah, ill instead check for if its not an empty object, nice catch